model/src/school/courses.ts

```diff
@@ -87,4 +95,11 @@ export namespace Course_functions {
 
 		createCourse: FunctionOf<{ courseJSON: string; }, { courseID: CourseID; }, 'cannot create the course'>("createCourse"),
 	});
-}
\ No newline at end of file
+}
+
+export type UpdateCourseArgs = {
+	name: string,
+	description: string,
+	credits: Credits,
+	section: Section,
+}
```

Comment:
It is already done in the firebase functions so no need to add checks here since onSanitizedCall and all the other checks inside already handle those cases. Also, the `CourseParameters.tsx` file does not allow the user to click on the Update Changes button if those properties are not satisfied so no need to do that.